src/guidellm/main.py

```diff
@@ -197,6 +197,7 @@ def generate_benchmark_report(
     max_requests: Union[Literal["dataset"], int, None],
     output_path: str,
     cont_refresh_table: bool,
+    backend_kwargs: Mapping[str, Any] = {},
```

Comment:
Can you redo this to avoid [mutable arguments](https://github.com/satwikkansal/wtfpython?tab=readme-ov-file#-beware-of-default-mutable-arguments)? Something like the following with a `None` check in the function.

```suggestion
    backend_kwargs: Optional[Mapping[str, Any]] = None,
```